decidim-admin/app/controllers/decidim/admin/officializations_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # Controller that allows managing user groups at the admin panel.
+    #
+    class OfficializationsController < Decidim::Admin::ApplicationController
+      layout "decidim/admin/users"
+
+      helper_method :user
+
+      def index
+        authorize! :index, :officializations
+        @query = params[:q]
+        @state = params[:state]
+
+        @users = Decidim::Admin::UsersOfficialization.for(@query, @state)
+                                                     .page(params[:page])
+                                                     .per(15)
+      end
+
+      def new
+        authorize! :new, :officializations
+
+        @form = form(OfficializationForm).from_model(user)
+      end
+
+      def create
+        authorize! :create, :officializations
+
+        @form = form(OfficializationForm).from_params(params)
+
+        OfficializeUser.call(@form) do
+          on(:ok) do |user|
+            notice = I18n.t("officializations.create.success", scope: "decidim.admin")
+
+            redirect_to officializations_path(q: user.name), notice: notice
```

Comment:
Other nit here. Note that I'm redirecting back to the list filtering by the user just officialized. This is because I find it... not sure, weird?, when I edit a record and I can't see it properly edited, back in the list. But this is just me? Or maybe we can use a better solution like sorting the list descending by the `updated_at` column?

Reply:
I understand the concern but current implementation is OK to me

Reply:
Good! We can iterate later if admins don't find this useful!